actual-rest-api/openapi.yaml

```diff
@@ -0,0 +1,142 @@
+openapi: 3.0.0
```

Comment:
The folder of the file is still our temporary thingy to avoid the name conflict with the old example. Maybe this can be sorted out here already?

Reply:
yes, will move it to `openapi/openapi.yaml`